controllers/gcpmachine_controller_test.go

```diff
@@ -17,12 +17,27 @@ limitations under the License.
 package controllers
 
 import (
+	"bytes"
+	"context"
+	"flag"
+
+	capierrors "sigs.k8s.io/cluster-api/errors"
```

Comment:
nit, this should be grouped with the imports below

Reply:
will change